src/interfaces/ICommand.ts

```diff
@@ -4,5 +4,5 @@ import { IMessage } from './IMessage';
 export interface ICommand {
   id: string;
   args: ICommandArgument[];
-  execute(message: IMessage, args: string[]): Promise<void>;
```

Comment:
Don't remove the return type; we need to enforce `execute` to always return a `Promise` of type `void`.

Reply:
This ^, this also means more things have to change in the command files.